draft-ietf-mls-architecture.md

```diff
@@ -1195,6 +1195,28 @@ may be possible to operate MLS in ways that provide certain deniability
 properties, but defining the specific requirements and resulting notions of
 deniability requires further analysis.
 
+### Associating a User's Clients
+
+When the same user uses multiple clients, it may be possible for other members
+of a group to recognize all of those clients as belonging to the same user.  For
+example, all of a user's clients might present credentials authenticating the
+user's identity.  This association among devices might be considered a leak of
+private information.  An application that wishes to mitigate this privacy risk
+has a couple of options.
+
+This risk only arises when the leaf nodes for the clients in question provide
+data that can be used to correlate the clients.  So one way to mitigate this
+risk is by only doing device-level authentication within MLS. If user-level
```

Comment:
Is this text trying to say that the credentials only contain device identity?

Reply:
This should be per client not per device.